sinks/pb/sinks_grpc.pb.go

```diff
@@ -0,0 +1,101 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
again I am confused why there are two generated pb.go files, I was pretty sure there was only one? also let's add them to Makefile